frontend/search/search_mobile.html

```diff
@@ -11,41 +11,57 @@
+            <md-input-container>
+                <md-select ng-model="searchCtrl.searchActuation" placeholder="ÁREA DE ATUAÇÃO" class="search-by-mobile">
+                    <md-option ng-repeat="actuation_area in searchCtrl.actuationAreas | orderBy: 'name'" ng-value="actuation_area.value"
+                        ng-click="searchCtrl.searchBy(actuation_area.value)">
+                        {{actuation_area.name}}
+                    </md-option>
+                </md-select>
+            </md-input-container>
+            <md-input-container>
+                <md-select ng-model="searchCtrl.searchNature" placeholder="NATUREZA LEGAL" class="search-by-mobile">
+                    <md-option ng-repeat="nature in searchCtrl.legalNature | orderBy: 'name'" ng-value="nature.value" ng-click="searchCtrl.searchBy(nature.value)">
+                        {{nature.name}}
+                    </md-option>
+                </md-select>
+            </md-input-container>
+            <md-input-container>
+                <md-select ng-model="searchCtrl.searchState" placeholder="LOCALIZAÇÃO" class="search-by-mobile">
+                    <md-option ng-repeat="state in searchCtrl.brazilianFederalStates" ng-value="state.nome" ng-click="searchCtrl.searchBy(state.nome)">
+                        {{state.nome}}
+                    </md-option>
+                </md-select>
+            </md-input-container>
+        </div>
+        <div layout="row" layout-align="end center" class="search-mobile-action-buttons">
+            <md-button md-colors="{background:'grey-500'}" ng-click="searchCtrl.leaveMobileSearchPage()">
+                CANCELAR
+            </md-button>
+            <md-button md-colors="{background:'teal-500'}" ng-click="searchCtrl.search()" ng-disabled="!searchCtrl.hasChanges">
+                PESQUISAR
+            </md-button>
+        </div>
     </div>
-    <div layout="row" layout-align="end center" class="search-mobile-action-buttons">
-        <md-button md-colors="{background:'grey-500'}" ng-click="searchCtrl.leaveMobileSearchPage()">
-            CANCELAR
-        </md-button>
-        <md-button md-colors="{background:'teal-500'}" ng-click="searchCtrl.search()"
-            ng-disabled="!searchCtrl.hasChanges">
-            PESQUISAR
-        </md-button>
+    <div ng-if="!searchCtrl.hasNotSearched">
+        <div class="registered-institutions-container" ng-if="searchCtrl.institutions.length > 1" style="margin-top: -1px; margin-bottom: 5px;">
+            <registered-institution ng-repeat="institution in institutions = (searchCtrl.institutions 
+                        | filter: searchCtrl.searchActuation | filter: searchCtrl.searchNature | filter: searchCtrl.searchState)" 
+                institution="institution" user="searchCtrl.user" style="margin-top: 10px;">
+            </registered-institution>
+        </div>
+        <div ng-if="searchCtrl.institutions.length === 1">
```

Comment:
Margin-left and rigth of the card content is bigger than it should.

![image](https://user-images.githubusercontent.com/7011394/53579881-2deac700-3b59-11e9-91bf-beec8de229e1.png)